resources/references/core-reference/dal-reference/aggregations-reference.md

```diff
@@ -46,7 +46,7 @@ $aggregation->getAvg();
 {% tab title="API Criteria" %}
 Request
 
-```javascript
```

Comment:
@trandangtri on the top of the file there is a table of all existing aggregations, please add the new aggregation as well there

Reply:
Thanks @keulinho . Just updated it.